.github/dependabot.yml

```diff
@@ -0,0 +1,126 @@
+version: 2
+updates:
+  - package-ecosystem: maven
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "08:00"
```

Comment:
I think there's an issue here. It was supposed to be in the evening before. I think it was 8pm or 9pm Paris time as defined in the Dependabot config UI.

And it just created a bunch of PRs for the BOM so something is not right :/

Reply:
> I think there's an issue here. It was supposed to be in the evening before. I think it was 8pm or 9pm Paris time as defined in the Dependabot config UI.


It was originally set to `21:00` and I set that to `08:00` on purpose, but I can change that if you prefer

> And it just created a bunch of PRs for the BOM so something is not right :/

I had the same impression, but then I realized it created 3 for each entry in `updates`, so I think that's correct.